From 0d2c28f097b4216ed8358c54b4d05d5b567fed4e Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sun, 7 Aug 2022 16:45:30 +0200
Subject: block: Do not include linux/fs.h any more

glibc 2.36 changed the definition of enum fsconfig_command, it now
collides with the same definition from sys/mount.h. Remove the include
of linux/fs.h This still compiled with musl too.

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 block.c | 2 --
 1 file changed, 2 deletions(-)

--- a/block.c
+++ b/block.c
@@ -33,8 +33,6 @@
 #include <sys/wait.h>
 #include <sys/sysmacros.h>
 
-#include <linux/fs.h>
-
 #include <uci.h>
 #include <uci_blob.h>
 
